mscore/seq.cpp

```diff
@@ -1750,7 +1750,15 @@ void Seq::setLoopSelection()
 
       if (score && score->selection().isRange()) {
             cs->setLoopInTick(score->selection().tickStart());
-            cs->setLoopOutTick(score->selection().tickEnd());
+            cs->setLoopOutTick(score->selection().tickEnd());      
+            }
+      
+      // add a dummy event to loop end if it is not already there
+      // this is to let the playback reach the end completely before starting again
+      if (!events.count(cs->loopOutTick().ticks())) {
+            NPlayEvent ev;
+            ev.setVelo(0);
```

Comment:
I would do `ev.setValue(META_EOT)`, since that is what this event is (end of track). 0 is `ME_INVALID` or `META_SEQUENCE_NUMBER`, neither of which this event is, really.

Reply:
Well, it's not really an end-of-track event either, because the user can select a portion in the middle of the score to be looped. I did a bit of research about MIDI events, but I'm still not sure about which event exactly should be used here. I thought that using an invalid event is the best option because this isn't really a MIDI event, it's just a dummy object, and is never written to a MIDI file. Maybe being more explicit about it could be better?

```suggestion
            ev.setValue(ME_INVALID);
```

Reply:
Oh, right... I was under the impression that this happened when the loop was right at the end of the score, no? If not, then you probably don't need to change anything.

Reply:
No, this happens always whenever the user selects a loop.